test/random/RadialDistribution.test.cu

```diff
@@ -40,7 +40,7 @@ struct RngStateView
 
 __global__ void sample(RngStateView view)
 {
-    int local_thread_id = celeritas::KernelParamCalculator::thread_id();
+    int local_thread_id = celeritas::KernelParamCalculator::thread_id().get();
```

Comment:
Technically, `ThreadID` is templated on an `unsigned int`.  Maybe to enforce "good" usage of opaque id types we should either explicitly use `unsigned id` here or just `auto`.

Reply:
Nice  catch, and you're right -- I'll change to  unsigned int, which is by CUDA definition the integer type underlying the `threadIdx` struct.